README.md

```diff
@@ -381,12 +381,106 @@ rer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" https://metrics.
 
 ### Writing Release Notes
 
-Release notes are maintained in the [change log](CHANGELOG.md) and follow guidelines
-based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). We include additional
-guidelines for increased readability and consistency:
-
-* Changes should be grouped by section, either as an enhancement, bug fix, or
-  internal change
-* References to code or configuration should render as literals
-* Bug fixes should link to bug reports (GitHub Issues or Jira items)
-* Features or enhancements should link to formal RFEs (GitHub Issues or Jira items)
+Release notes are maintained in the [changelog](CHANGELOG.md) and follow
+guidelines based on [keep a changelog](https://keepachangelog.com/en/1.0.0/).
+This section describes additional guidelines, conventions, and the overall
+process for writing release notes.
+
+#### Guidelines
+
+* Each release should contain release notes
+* Changes should be applicable to at least one of the six types listed below
+* Use literals for code and configuration
+  - ``defaultScanSettingsSchedule``
+  - ``nodeSelector``
+* Write your notes with users as the audience
+* Keep notes as short as possible and link to additional documentation when necessary
+  - Bug fixes should link to bug reports (GitHub Issues or Jira items)
+  - Features or enhancements should link to formal RFEs (GitHub Issues or Jira
+    items)
+
+#### Change Types
+
+The following describe each potential section for a release changelog.
+
+1. Enhancements
+2. Fixes
+3. Internal Changes
+4. Deprecations
+5. Removals
+6. Security
+
+*Enhancements* are reserved for communicating any new features or
+functionality. You should include any new configuration or processes a user
+needs to take to use the new feature.
+
+*Fixes* are for noting improvements to any existing functionality.
+
+*Internal Changes* are ideal for communicating refactors not exposed to end
+users. Even if a change does not directly impact end users, it is still
+important to highlight paying down technical debt and the rationale for those
+changes, especially since they impact the project's roadmap.
+
+*Deprecations* is for any functionality, feature, or configuration that is
+being deprecated and staged for removal. Deprecations should include why we're
+preparing to remove the functionality and signal any suitable replacements
+users should adopt.
+
+*Removals* is for any functionality, feature, or configuration that is being
+removed. Typically, entries in this section will have been deprecated for some
+period of time.
+
+*Security* is reserved for any security-related change, or relevant to a
+security-focused audience. Readers focused on only the security-specific
+content for a release should be able to glean all the information from this
+section.
```

Comment:
Hmm, interesting, is this common? I thought that usually the security part is where the CVE fixes are called out. But I don't mind either way, just asking.

Reply:
I've seen some projects use it to communicate security-related changes outside of CVEs, but I can back out this specific description and stick to the one documented the Keep A Changelog reference [0].

[0] https://keepachangelog.com/en/1.0.0/#how